examples/flexvolume/nginx.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    volumeMounts:
+    - name: test
+      mountPath: /data
+    ports:
+    - containerPort: 80
+  volumes:
+  - name: test
+    flexVolume:
+      driver: "kubernetes.io/lvm"
+      fsType: "ext4"
+      options:
+        volumeID: "vol1"
+        size: "1000m"
+        volumegroup: "kube_vg"
+        fsType: "ext4"
```

Comment:
Only mounting code should care about `fstype`, since the callouts currently only do attach/detach `fstype` should be irrelevant to them and not passed to them.